### DISCUSSION TITLE
Gifs for github pages?

### DISCUSSION THREAD

**Discussion by rne1223** [nmbr73/Shadertoys#22] [Ideas]
Just wondering if you would be open to the idea of using gifs instead of pictures for the github page. 

With that being said, I just opened this discussion to get a feel of how it works. Keep up the great work guys.

@nmbr73 @J-i-P-i

**Comment by J-i-P-i** [comment]
In fact, at the beginning I created little gifs ( https://github.com/nmbr73/Shadertoys/blob/main/PlanetShader/CrackerCars.md ) 
It just makes "work" - but I will do it for particularly beautiful ones :
![DiffuisonGathering](https://user-images.githubusercontent.com/78935215/109801188-72187280-7c1e-11eb-9487-e82009aedd7a.gif)
